tests/integration/tests/test-securityTips-page.js

```diff
@@ -8,6 +8,6 @@ describe("Firefox Monitor Security Tips Page", function() {
   });
 
   it("should look like normal", function() {
-    expect(browser.checkFullPageScreen("SecurityTips_Page", {})).to.be.within(0, 5);
+    expect(browser.checkFullPageScreen("SecurityTips_Page", {})).to.be.within(0, 15);
```

Comment:
Please add ```ignoreAntialiasing: true``` to the array

Reply:
@jrbenny35 And just to remind myself, the `.within(0, 15)` means that 0-15% of the pixels in the screenshot can be wrong (versus the baseline image)?

Reply:
Yes. 15% of the overall page can be wrong. Since this is a big page, its a big percentage of that page.